README.md

```diff
@@ -147,6 +147,19 @@ Add the following to your `pom.xml`:
 </build>
 ```
 
+And it's dependency:
+
+```xml
+<dependencies>
+    <dependency>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-assembly-plugin</artifactId>
+        <version>3.3.0</version>
+        <type>maven-plugin</type>
+    </dependency>
+</dependencies>
```

Comment:
This doesn't make much sense to me. You shouldn't need to add maven assembly plugin as a dependency, is this a new thing in maven or something? This information is usually described in the `<plugins>` tag rather than `<dependencies>`